catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -543,7 +546,7 @@ protected DAGQueryResult getData(long queryTime) {
 
             QueryRequestImpl catalogQueryRequest;
             if (querySources == null || querySources.isEmpty()) {
-                LOGGER.trace("Query request will be local, no sources specified");
+                LOGGER.trace("Query request will be local, no sources specified: {}", parsedFilter);
```

Comment:
Depending on the filter. The Temporal Filter does not have a good toString(), but the text comparison ones do.